packages/ui-extensions/src/surfaces/customer-account/api/docs.ts

```diff
@@ -77,3 +78,49 @@ export interface Docs_Standard_QueryApi
 export interface Docs_StandardApi extends Omit<StandardApi<any>, 'router'> {}
 
 export interface Docs_FullPageApi extends Omit<FullPageApi, 'location'> {}
+
+export interface Docs_Page_Button_PrimaryAction
+  extends Pick<
+    ButtonProps,
+    | 'onPress'
+    | 'to'
+    | 'loading'
+    | 'loadingLabel'
+    | 'disabled'
+    | 'accessibilityLabel'
+  > {}
+export interface Docs_Page_Button_SecondaryAction
+  extends Pick<
+    ButtonProps,
+    'onPress' | 'to' | 'loading' | 'loadingLabel' | 'disabled'
```

Comment:
I don't think we're adding `loading`, `loadingLabel`, `disabled` to the secondaryAction (i.e. back button).

Reply:
good catch! fixed.